exersice_html/style.css

```diff
@@ -0,0 +1,341 @@
+  padding-top: 26px;
+}
+.banner-img {
+  width: 100%;
+  background-size: cover;
+}
+.toolbar {
+  padding: 30px 16px 40px 16px;
+  display: flex;
+  justify-content: space-between;
+}
+.sort {
+  position: relative;
+}
+.sort > select{
+  border-radius: 4px;
+  padding-left: 12px;
+  font-size: 14px;
+  height: 36px;
+}
+.sort-uselessFirst {
+  width: 142px;
+  border: 1px solid #5A5B6A;
+  font-weight: 500;
+  color: #19191D;
+}
+.sort::before {
+  position: absolute;
+  top: -5px;
+  left: 12px;
+  content: "SORT BY";
+  font-size: 10px;
+  font-weight: 500;
+  color: #787885;
+  background-color: #fff;
+  font-family: 'inter', sans-serif;
+}
+.sort-condition {
+  width: 118px;
+  border: 1px solid #B4B4BB;
+  font-weight: 400;
+  color: #787885;
+}
+.sort-delivery {
+  width: 158px;
+  border: 1px solid #B4B4BB;
+  font-weight: 400;
+  color: #787885;
+}
+.action {
+  display: flex;
+}
+.action-group, .type-show {
+  border-radius: 4px;
+  background-color: #EBF2FF;
+  display: flex;
+}
+.action-btn >input {
+  display:none;
+}
+.action-btn >label {
+  background-color:#EBF2FF;;
+  font-size: 14px;
+  font-weight: 400;
+}
+.action-btn {
+  line-height: 36px;
+  width: 85px;
+  height: 36px;
+  font-size: 14px;
+  font-weight: 400;
+  background-color: #EBF2FF;
+}
+.action-btn:nth-child(2),
+.type-show-btn:nth-child(2) {
+  background-color: #fff;
+  box-shadow: 0px 1px 2px rgba(27, 78, 163, 0.24);
+}
+.action-btn:nth-child(2) >label {
+  background-color: #fff;
+}
+.type-show {
+  margin-left: 24px; 
+}
+.type-show .action-btn {
+  width: 68px;
+}
+.action-btn .fa.fa-list {
+  color: #787885;
+  font-size: 17px;
+}
+.action-btn .fa.fa-th-large {
+  color: #2979FF;
+  font-size: 18px;
+}
+.grid-list {
+  display: flex;
+  flex-wrap: wrap;
+}
+.products {
+  margin-bottom: 104px;
+  padding: 0 11px;
+}
+.grid-item {
+  width: calc(25% - 10px);
+  margin: 0 5px 16px 5px;
+  padding: 16px 16px 12px 16px;
+  border-radius: 8px;
+}
+.product{
+  transition: all 0.5s;
+}
+.product:hover {
+  box-shadow: 0px 4px 8px rgba(90, 91, 106, 0.2);
+}
+.product-content {
+  font-style: normal;
+  font-size: 16px;
+  font-weight: 400;
+  color:#19191D;
+  margin-top: 12px;
+  height: 74px;
+}
+.product-price {
+  font-size: 24px;
+  font-weight: 700;
+  color: #000;
+}
+.product-detail {
+  height: 41px;
+  font-size: 14px;
+  font-weight: 400;
+  margin-top: 8px;
+  margin-bottom: 9px;
+  color:#787885;
+}
+.btn-add-cart {
+  line-height: 20px;
+  padding: 4px 12px;
+  font-weight: 500;
+  font-size: 14px;
+}
+.product-info-group {
+  margin-top: 8px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+.product-extra {
+  width: 70px;
+  height: 24px;
+  line-height: 24px;
+  font-size: 12px;
+  font-weight: 400;
+}
+.product-cool-deal {
+  background-color: #FDEDF2;
+  color: #C23564;
+}
+.product-sale {
+  background-color: #ECF7ED;
+  color: #37833B;
+}
+.page-footer {
+  background-color:#2264D1;
+  height: 275px;
```

Comment:
Consider to set fixed height for a element. The height will be scaled if we do css correctly for inside content. When do responsive on mobile, we resize font-size, margin... of the child elements, the height is resize to small too